reference-architecture/day2ops/scripts/project_export.sh

```diff
@@ -31,48 +31,60 @@ usage(){
   warnuser
 }
 
-ns(){
-  echo "Exporting namespace to ${PROJECT}/ns.json"
-  oc get --export -o=json ns/${PROJECT} | jq '
-    del(.status,
-      .metadata.uid,
-      .metadata.selfLink,
-      .metadata.resourceVersion,
-      .metadata.creationTimestamp,
-      .metadata.generation
-      )' > ${PROJECT}/ns.json
+exportlist(){
+  if [ "$#" -lt "3" ]; then
+    echo "Invalid parameters"
+    return
+  fi
+
+  KIND=$1
+  BASENAME=$2
+  DELETEPARAM=$3
+
+  echo "Exporting '${KIND}' resources to ${PROJECT}/${BASENAME}.json"
+
+  BUFFER=`oc get ${KIND} --export -o json -n ${PROJECT} || true`
```

Comment:
Do you mind to use $() instead ``? mywiki.wooledge.org/BashFAQ/082

Reply:
Sorry, fixed in 02cfd39.